e2e-testing/test-ws/.vscode/settings.json

```diff
@@ -1 +1,4 @@
-{}
+{
```

Comment:
This is an attempt to deal with some flaky tests that may be related to VS Code prompts showing over a panel